README.md

```diff
@@ -517,14 +517,53 @@ Specifies the file extension for **local** script files in your Apps Script proj
 
 Specifies the files that should be pushed first, useful for scripts that rely on order of execution. All other files are pushed after this list of files.
 
+## Using clasp as a local dependency
+
+Typically `clasp` is installed as a global package for convinience when using the command line.
+
+Still there may be occassions when you want a project to use a specific version of `clasp` (i.e. to prevent potential breaking changes in future version)
+
+This can be achieved by simply adding clasp as a local dependency and a fixed version
+
+```sh
+npm install --save-exact @google/clasp
+```
+
+Optionally, you can specify an older version.
+
+```sh
+npm install --save-exact @google/clasp@1.7.0
+```
+
+After this, any npm scripts using `clasp`will use the **local** version
+
+```json
+  "scripts": {
+    "push-project1": "cd src/GasProject1 && clasp push",
+  },
+```
+
+From the command line, within your project, use the `npx` tool in order to use the **local** version of `clasp` rather than any global one.
+
+```sh
+npx clasp --version
+```
+
 ## Troubleshooting
 
-### Node Version
+### NodeJS Version
```

Comment:
The main README needs less text, not more.

I oppose this change.

Reply:
Just to be sure I get it: Node is préfères over NodeJS right? If so, I will revert.

Reply:
No, this whole section should be reverted. There are no useful changes.

Reply:
I beg to differ ;) supported version of node is now up to date. And the upgrade instruction no longer "pollutes" your global node_modules with a "one-shot" `n` module.
I would personally drop the whole paragraph, on the reasoning that checking and upgrading node version is already covered length on SO and the rest of the internet.